main/exercise/admin.php

```diff
@@ -160,6 +160,11 @@
     api_not_allowed(true);
 }
 
+if ($objExercise->selectPtType() == EXERCISE_PT_TYPE_PTEST) {
+    header('Location: '.api_get_path(WEB_CODE_PATH).'exercise/ptest_admin.php?exerciseId='.$exerciseId.'&'.api_get_cidreq());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 125 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)